### DISCUSSION TITLE
Netbox and duplicate records

### DISCUSSION THREAD

**Discussion by john2893** [netbox-community/netbox#14449] [Help Wanted!] [answered]
### why does netbox allow duplicate records to be created? 
I created so many if and else conditions to check and left the A records check as it is because A records when created again said already exists. 
Now A records gets created even if it exists. 
I created a check of TXT, NS, MX and SRV.  I'm pretty sure this issue would have been solved long back.
Anyone help me with some suggestions please.

**Comment by jeremystretch** [comment, marked as answer]
NetBox does not manage DNS records. Are you referring to the [netbox-dns](https://github.com/auroraresearchlab/netbox-dns) plugin perhaps? You'll need to raise this concern with the plugin you're using.
